2022-02-20T14:00:16.540+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=account_json": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-02-20T14:00:16.541+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=account_json": dial tcp 192.168.1.4:8848: connect: no route to host> , result:<>
2022-02-20T14:00:16.541+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down> , result:<>
2022-02-20T14:00:16.542+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down 
2022-02-20T14:00:16.542+0800	[31mERROR[0m	get config from cache  error:failed to read config cache file:nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b,err:open nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b: no such file or directory  
2022-02-20T14:05:13.496+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-02-20T14:05:13.497+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: no route to host> , result:<>
2022-02-20T14:05:13.497+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down> , result:<>
2022-02-20T14:05:13.497+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down 
2022-02-20T14:05:13.497+0800	[31mERROR[0m	get config from cache  error:failed to read config cache file:nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b,err:open nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b: no such file or directory  
2022-02-20T14:06:15.513+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-02-20T14:06:15.513+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: no route to host> , result:<>
2022-02-20T14:06:15.514+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down> , result:<>
2022-02-20T14:06:15.514+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.4:8848: connect: host is down 
2022-02-20T14:06:15.514+0800	[31mERROR[0m	get config from cache  error:failed to read config cache file:nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b,err:open nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b: no such file or directory  
2022-02-20T14:09:14.357+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?dataId=account_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-02-20T14:09:14.358+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=account_json": dial tcp 192.168.1.4:8848: connect: no route to host> , result:<>
2022-02-20T14:09:14.359+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"account_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.4:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=account_json": dial tcp 192.168.1.4:8848: connect: host is down> , result:<>
2022-02-20T14:09:14.359+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.4:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=account_json": dial tcp 192.168.1.4:8848: connect: host is down 
2022-02-20T14:09:14.359+0800	[31mERROR[0m	get config from cache  error:failed to read config cache file:nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b,err:open nacos/cache/config/account_json@@dev@@dc7c4c11-eff2-4175-a977-50fe4b0ccd7b: no such file or directory  
2022-03-06T01:49:00.112+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-03-06T01:49:00.114+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: no route to host> , result:<>
2022-03-06T01:49:00.114+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: host is down> , result:<>
2022-03-06T01:49:00.114+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: host is down 
2022-03-06T01:49:32.152+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b&dataId=product_json": context deadline exceeded (Client.Timeout exceeded while awaiting headers)> , result:<>
2022-03-06T01:49:32.153+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: no route to host> , result:<>
2022-03-06T01:49:32.154+0800	[31mERROR[0m	api</v1/cs/configs>,method:<GET>, params:<{"dataId":"product_json","group":"dev","tenant":"dc7c4c11-eff2-4175-a977-50fe4b0ccd7b"}>, call domain error:<Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: host is down> , result:<>
2022-03-06T01:49:32.154+0800	[31mERROR[0m	get config from server error:Get "http://192.168.1.2:8848/nacos/v1/cs/configs?dataId=product_json&group=dev&tenant=dc7c4c11-eff2-4175-a977-50fe4b0ccd7b": dial tcp 192.168.1.2:8848: connect: host is down 
